src/examples/databases/src/backend/schema/user/entity.ts

```diff
@@ -1,5 +1,4 @@
-import { GraphQLEntity, SummaryField } from '@exogee/graphweaver';
-import { Field, ID, ObjectType } from 'type-graphql';
+import { GraphQLEntity, SummaryField, Field, ID, ObjectType } from '@exogee/graphweaver';
```

Comment:
All of these "Field, ID, ObjectType, Resolvers" etc are they exactly the same thing just coming through gw now?

Reply:
yeah, I did this for two reasons:

1. one less thing to explain to a new user
2. If we wanted to overwrite one with our own custom version we could and there would be no client changes needed.

what do you think?

Reply:
Yup that makes sense to me!